assets/src/blocks/amp-story-post-author/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+
+export const name = 'amp/amp-story-post-author';
+
+export const settings = {
+	title: __( 'Story Author', 'amp' ),
```

Comment:
Maybe for author it would be good to set a bit different defaults, for example, smaller font size and not using AMP Fit Text. Same for the date perhaps. Thoughts?

Reply:
Yeah adding some better defaults makes sense. I haven't added styling yet, so it should be easy.

I thought I had disabled amp-fit-text for all the meta blocks, thinking that the user might want to always have a fixed size for them as the content never really changes, which makes it easy to set the right size once.

Reply:
Improved styling a bit and removed the initial font size which I copied over from the text block.